hack/update-import-aliases.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+SCRIPT_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
+cd "${SCRIPT_ROOT}"
+
+ROOT_PATH=$(pwd)
+GOPATH=$(go env GOPATH | awk -F ':' '{print $1}')
+
+temp_path=$(mktemp -d)
+pushd "${temp_path}" >/dev/null
+cp "${ROOT_PATH}"/go.mod .
```

Comment:
Why copy the `go.mod`?
Can we just install the `preferredimports` by command:
```
go get k8s.io/kubernetes/cmd/preferredimports@v1.21.3
```

Reply:
on my side `go get k8s.io/kubernetes/cmd/preferredimports@v1.21.3`  directly is unsuccessful and will report an error:
`go get: k8s.io/kubernetes@v1.21.3 requires
        k8s.io/api@v0.0.0: reading k8s.io/api/go.mod at revision v0.0.0: 
`

Reply:
Get it. That's the reason why you copy the `go.mod`. That's fine.